### PR TITLE
R103

### PR DESCRIPTION
**Pull request by chuwy**
/cc @mhadam

### CODE REVIEW

**Comment by alexanderdean** [comment]
Has this been reviewed?

**Comment by chuwy** [comment]
Just by me. @BenFradet can you also have a look?

File: schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            "type": "string",
+            "description": "The name of the screen viewed."
+        },
+        "type": {
+            "type": "string",
+            "description": "The type of screen that was viewed e.g feed / carousel."
+        },
+        "id": {
+            "type": "string",
+	    "format": "uuid",
```
Comment (BenFradet) [inline schemas/com.snowplowanalytics.mobile/screen/jsonschema/1-0-0]: :art: indent

File: schemas/com.snowplowanalytics.mobile/screen_view/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            "type": "string",
+            "description": "The name of the screen viewed."
+        },
+        "type": {
+            "type": "string",
+            "description": "The type of screen that was viewed e.g feed / carousel."
+        },
+        "id": {
+            "type": "string",
+	    "format": "uuid",
```
Comment (BenFradet) [inline schemas/com.snowplowanalytics.mobile/screen_view/jsonschema/1-0-0]: :art:

File: schemas/com.snowplowanalytics.mobile/screen_view/jsonschema/1-0-0 [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            "maxLength": 36,
+            "description": "An ID from the associated screenview event."
+        },
+        "previousName": {
+            "type": "string",
+            "description": "The name of the previous screen."
+        },
+        "previousId": {
+            "type": "string",
+	    "format": "uuid",
```
Comment (BenFradet) [inline schemas/com.snowplowanalytics.mobile/screen_view/jsonschema/1-0-0]: :art:

File: schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0 [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+	"description": "Schema for an event where a mobile application is installed.",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "application_install",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+	},
```
Comment (BenFradet) [inline schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0]: just a q, what's the point of a schema without properties?

**Comment by alexanderdean** [comment]
Not sure I follow - it has feedback from @BenFradet but has been marked as approved?

**Comment by BenFradet** [comment]
it's just minor stuff that do not require a re-review from me

**Comment by alexanderdean** [comment]
Okay back to @chuwy to fix. Please re-assign to me when ready for publication.

File: schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0 (hunk 4 above)
Comment (chuwy) [reply schemas/com.snowplowanalytics.mobile/application_install/jsonschema/1-0-0]: Assuming its `Map[String, Any]`. Technically, `patternProperties` could express it better, but having few troubles with `patternProperties` in Iglu - I now tend to advise against it.

**Comment by chuwy** [comment]
Done, @alexanderdean.

File: CHANGELOG [hunk 5]
```diff
@@ -1,3 +1,9 @@
+Release 103 (2019-05-03)
+------------------------
+Add com.snowplowanalytics.mobile/screen/jsonschema/1-0-0 (#866)
+Add com.snowplowanalytics.mobile/screenview/jsonschema/1-0-0 (#919)
```
Comment (alexanderdean) [inline CHANGELOG]: Event is called screen_view not screenview

File: schemas/com.snowplowanalytics.mobile/screen_view/jsonschema/1-0-0 [hunk 6]
```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a screenview event",
```
Comment (alexanderdean) [inline schemas/com.snowplowanalytics.mobile/screen_view/jsonschema/1-0-0]: Event is called screen_view not screenview

File: CHANGELOG (hunk 5 above)
Comment (chuwy) [reply CHANGELOG]: Fixed!

**Pull request merged**